debian-mk-kernel.mk

```diff
@@ -24,12 +28,14 @@ bbl-q: $(KERNEL_IMAGE)
 	rm -rf riscv-pk/build
 	mkdir -p riscv-pk/build
 	cd riscv-pk/build && \
+	PATH="$(CROSS_COMPILE_DIR):${PATH}" \
 	../configure \
-	    --host=riscv64-linux-gnu \
+	    --host=${PREFIX} \
```

Comment:
I do not really understand why we need  `${PREFIX}` - this specifies the arch/os which is always going to be `riscv64-linux-gnu`. 
Besides, please use make variables (`$(PREFIX)`) than passing it unexpanded down to shell and depending on shell picking up the value from environment.

Reply:
PREFIX allows to choose toolchain. Linux kernel can be built by variable toolchains not only via riscv64-linux-gnu. In embedded projects riscv64-unknown-elf is used very often and my team use too.

Reply:
Right, but then is should not be called  `PREFIX` but rather `KERNEL_HOST` no?

Reply:
PREFIX is used because PREFIX-gcc. prefix is first part of name file. And there is POSTFIX which is last part of name.

Reply:
If you think your name of variable is more informable I will change

Reply:
Yes, I think so. Thanks!

Reply:
done